src/v/kafka/server/handlers/topics/types.h

```diff
@@ -46,6 +46,12 @@ static constexpr std::string_view topic_property_retention_bytes
 static constexpr std::string_view topic_property_retention_duration
   = "retention.ms";
 
+// Data-policy property
+static constexpr std::string_view topic_property_data_policy_function_name
+  = "redpanda.datapolicy.function.name";
+static constexpr std::string_view topic_property_data_policy_script_name
```

Comment:
@dotnwat 
For set data-policy user need to send set `redpanda.datapolicy.function.name` and  `redpanda.datapolicy.script.name` in one update command
For delete user need to do the same logic with remove command for topic properties (for both function_name and script_name)

Maybe we need to add new topic property (redpanda.datapolicy) wich user can be used only for remove, and it will remove function_name and script_name. But user can not set this property. Only for simple delete command

Reply:
I think we should keep things as simple as possible for now, and creating a new topic property that's only meaningful when deleting seems complicated, but I admit, I don't fully understand your proposal. what is it about deleting both properties at the same time that is problematic?

here is an alternative: like we removed the json, we could also remove the two topic properties and use a single property. let's say its `redpanda.datapolicy` and then the format is `script_name::function_name`. then to delete the policy you can delete the property or set it to `""` or `null` or whatever makes sense. wdyt?

Reply:
I do not think it (delete 2 properties in one request) can be problem 

I only wanted to add this suggestion